horace_core/admin/check_horace_mex.m

```diff
@@ -35,8 +35,8 @@
     'compute_pix_sums  : ', ...
     'combine_sqw       : ', ...
     'mtimesx_mex       : ', ...
-    'hdf_mex_reader    : ', ...
-    'GetMD5            : '  ...
+    'GetMD5            : ',  ...
```

Comment:
Nothing wrong here but it is not clear why these lines have to be swapped. a comment saying why would clarify this for future developers. thanks

Reply:
hdf mex reader is not used, does not work for current versions of Matlab and should probably be dropped.

I spent a lot of time on it but do not see it usage in any forceable future, so I am dropping it now.